gradle/jmh.gradle

```diff
@@ -0,0 +1,81 @@
+apply plugin: 'java'
+apply plugin: 'idea'
+
+project.sourceSets.create('jmh') {
+    java.srcDir 'src/jmh/java'
```

Comment:
I think we can do this without the source-set hackery, I added similar benchmarks to auth-tokens a while back and I think it was simpler

Reply:
i removed this file, i keep jmh in the test path.